packages/utils/src/route.js

```diff
@@ -80,15 +80,14 @@ function cleanChildrenRoutes (routes, isChild = false, routeNameSplitter = '-',
     }
     route.name = route.name.replace(regExpIndex, '')
     if (route.children) {
-      const indexRoutePath = trailingSlash === false ? '/' : ''
-      const defaultChildRoute = route.children.find(child => child.path === indexRoutePath)
+      const defaultChildRoute = route.children.find(child => child.path === '/' || child.path === '')
       const routeName = route.name
       if (defaultChildRoute) {
         if (trailingSlash === false) {
```

Comment:
I think we can remove this condition now. We are going to remove `route.name` therefore we can use its name for the default child